py-polars/polars/series/utils.py

```diff
@@ -39,7 +39,12 @@ def expr_dispatch(cls: type[T]) -> type[T]:
     expr_lookup = _expr_lookup(namespace)
 
     for name in dir(cls):
-        if not name.startswith("_"):
+        if (
+            # private
+            not name.startswith("_")
+            # `.plot` not available on Expr
+            and name != "plot"
```

Comment:
This shouldn't be required - `plot` is not an empty function so it's skipped. Everything runs fine when I comment this out - am I missing something?

Reply:
did you try building the docs?

Reply:
Hm no, indeed it doesn't build the docs. But the problem isn't that `.plot` is not available on Expr. The problem is that it calls `hvPlotTabularPolars` which imports polars, which doesn't exist while building the docs.

I'm not sure of the best solution here. I want to avoid hardcoding random stuff here as the functionality is already complex enough. But I'm not sure what the best solution would be 🤔

Reply:
True, but my understanding is that `expr_dispatch` is only needed for methods available on `Expr`

you're right about complexity, I just couldn't think of a solution which doesn't add any. maybe there's a more generic solution which would mean that this isn't necessary

cc @alexander-beedie in case you have ideas here